src/hooks/sdk/docs/useDeviceLabelTriggerStatus.mdx

```diff
@@ -2,7 +2,7 @@
 
 # useDeviceLabelTriggerStatus
 
-The `useDeviceLabelTriggerStatus` hook returns the `DeviceLabelTrigger` status from the enum below.
+The `useDeviceLabelTriggerStatus` hook returns the [`DeviceLabelTrigger`](https://github.com/aws/amazon-chime-sdk-js/blob/main/src/devicecontroller/DeviceControllerFacade.ts#L25) status from the enum below.
```

Comment:
Suggest using the hosted link over the code link as code line number can change:
https://aws.github.io/amazon-chime-sdk-js/interfaces/devicecontrollerfacade.html#setdevicelabeltrigger

Reply:
Updated to permalink.